pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
     "Microsoft.CSharp": "4.3.0",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.3-servicing-25319-01",
+    "Microsoft.NETCore.Runtime.CoreCLR": "2.0.0-preview2-25319-01",
```

Comment:
@dagood Why is 1.1.0 servicing update getting 2.0.0-preview2 updates instead of 1.1.3-servicing?

CC @eerhardt @weshaggard